python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -136,8 +136,7 @@ def test_duplicated_column_names(self):
 
     def test_help_command(self):
         # Regression test for SPARK-5464
-        rdd = self.sc.parallelize(['{"foo":"bar"}', '{"foo":"baz"}'])
-        df = self.spark.read.json(rdd)
+        df = self.spark.createDataFrame([Row(foo="bar"), Row(foo="baz")])
```

Comment:
We should not replace like this because this is a regression test for SPARK-5464 . Please see the JIRA.